python/orca/dev/test/run-pytests-ray.sh

```diff
@@ -45,7 +45,8 @@ python -m pytest -v test/bigdl/orca/learn/ray \
       --ignore=test/bigdl/orca/learn/ray/pytorch/test_estimator_horovod_backend.py \
       --ignore=test/bigdl/orca/learn/ray/pytorch/test_estimator_ray_runtime.py \
       --ignore=test/bigdl/orca/learn/ray/pytorch/test_estimator_ray_dataset.py \
-      --ignore=test/bigdl/orca/learn/ray/tf/
+      --ignore=test/bigdl/orca/learn/ray/tf/ \
+      --ignore=test/bigdl/orca/learn/ray/mxnet/
```

Comment:
WIP: need to add mxnet elsewhere

Reply:
Refer to other jobs to add if condition for py37; we can only run mxnet for py37